### PR TITLE
use relative path, update gitignore, stop password managers from dete…

### PR DESCRIPTION
**Pull request by MichaelXian**
…cting membership field

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import App from "./App.tsx";
-import "G:/ReadMei/src/scss/style.scss";
```
Comment (MichaelXian) [inline src/main.tsx:5]: Using relative path so it doesn't matter where you store the project in your computer

File: src/reader.tsx [hunk 2]
```diff
@@ -39,9 +38,9 @@ function Input() {
             <input
               autoFocus
               ref={inputRef}
-              type="password"
+              type="text"
```
Comment (MichaelXian) [inline src/reader.tsx:41]: Using "text" input type stops password managers from saving the value. Keeping the functionality of hiding the text using css associated with hidden-text

File: tsconfig.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "useUnknownInCatchVariables": false,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true
```
Comment (MichaelXian) [inline tsconfig.json:23]: Not entirely sure what this does but things were red/not resolving in my IDE then I clicked a button and it added this which fixed it lmao

File: src/main.tsx (hunk 1 above)
Comment (itspoff) [reply src/main.tsx:5]: oh right thank you i forgor

**Review by itspoff — APPROVED** [review]
lgtm

**Pull request merged**